packages/language-server/scripts/update-diagnostics-snapshots.js

```diff
@@ -3,6 +3,7 @@ const klawSync = require("klaw-sync");
 const { forEach, filter, map } = require("lodash");
 const { resolve, dirname } = require("path");
 const { writeFileSync } = require("fs");
+const { existsSync } = require("fs-extra");
```

Comment:
`existsSync` is part of "fs" no need to import it from fs-extra.
- `fs-extra` is an automatic wrapper to `fs`